common/src/main/kotlin/com/artillexstudios/axminions/minions/Minion.kt

```diff
@@ -164,6 +166,10 @@ class Minion(
     }
 
     private fun breakMinion(event: PacketEntityInteractEvent) {
+        val preBreakEvent = MinionPreBreakEvent(event.player, this)
+        Bukkit.getPluginManager().callEvent(preBreakEvent)
+        if(preBreakEvent.isCancelled) return
```

Comment:
Could you add a space between the if and the "("? I'd appreciate if you made this change in the other places aswell.